src/main/scala/io/github/pauljamescleary/petstore/domain/orders/OrderRepositoryAlgebra.scala

```diff
@@ -1,12 +1,15 @@
 package io.github.pauljamescleary.petstore.domain.orders
 
+import cats.effect.Bracket
+
 import scala.language.higherKinds
 
 trait OrderRepositoryAlgebra[F[_]] {
+  type B = Bracket[F, Throwable]
```

Comment:
Instead of adding the explicit type signature everywhere, I just add this type member. Needing this implicit value at this level seems wrong. Other implementations of this trait will not have this restriction. The in-memory interpreters for example force their users to create these implicit brackets even though they're unnecessary. 

So while this works, it's not great.

Reply:
Hmm, yea this feels kind of odd.  I wonder if @tpolecat or any others can take a looksie

Reply:
I could get away from this by requiring the `implicit Bracket` at the class definition level, like `class OrderRepository[F[_]](implicit B: Bracket[F, Throwable]) extends OrderRepositoryAlgebra[F]`, but I don't know enough yet about `Bracket` to know if that works. Could users have a different bracket at the call site for running a query?

Reply:
Ok, I moved the bracket requirement to the class definition level, so the algebras are not polluted with it